config/test.exs

```diff
@@ -21,5 +21,6 @@ config :coney,
     timeout: 1000
   },
   workers: [
-    FakeConsumer
+    Coney.FakeConsumer,
```

Comment:
do we have to define this config again in test.exs if it already exists in config.exs?

Reply:
You are correct, it's duplicated, I didn't notice 👍